docs/_layouts/docs.html

```diff
@@ -155,6 +155,11 @@
         </a>
       </li>
       <li>
+        <a href="{{ site.baseurl }}/docs/external-volumes.html">
+          Stateful Applications Using External Volumes
```

Comment:
Are those called "external" or "persistent" volumes? The json schema in https://github.com/mesosphere/marathon/pull/3525 suggests the second. IMO "persistent" is the better term.